biofefi/options/enums.py

```diff
@@ -112,7 +112,7 @@ class SvmKernels(StrEnum):
 
 
 class Normalisations(StrEnum):
-    Standardization = "standardization"
+    Standardisation = "standardisation"
```

Comment:
```suggestion
    Standardisation = "standardization"
```
Keep the field value them for backwards compatibility